onnxruntime/contrib_ops/cpu/quantization/dynamic_quantize_matmul.cc

```diff
@@ -161,6 +161,115 @@ class DynamicQuantizeMatMul final : public MatMulIntegerToFloatBase {
 
   Status Compute(OpKernelContext* context) const override;
 
+  Status PrePack(const Tensor& tensor, int input_idx, AllocatorPtr alloc,
+                 /*out*/ bool& is_packed,
+                 /*out*/ PrePackedWeights* prepacked_weights) override {
+    // only pack Matrix B
+    if (input_idx == GetBIdx()) {
+      const Tensor* b_zp_tensor{nullptr};
+
+      // assume zero is present
+      auto b_zp_present{true};
```

Comment:
According to the spec of `DynamicQuantizeMatmul`, zero point is optional - https://github.com/microsoft/onnxruntime/blob/46c3b2c37e3154ef56e90c8fd3683c1d75b95073/onnxruntime/core/graph/contrib_ops/quantization_defs.cc#L395. Can/Should the code account for the case where it is missing ? (I think it is an enhancement that can come later as well).